onnxruntime/core/optimizer/matmul_transpose_fusion.cc

```diff
@@ -118,11 +114,13 @@ static size_t UpdateConsumerCount(Graph& graph, NodeArg* target, std::unordered_
 *                              |
 *                              V
 */
-static Node* GetTransposeNodeFromCast(Graph& graph, Node* cast) {
+static Node* GetTransposeNodeFromCast(Graph& graph, Node* cast,
```

Comment:
>GetTransposeNodeFromCast [](start = 13, length = 24)

since this is transform or swapping the nodes, I wonder the method name could be more intuitive

Reply:
This name is along the same form as another function previously defined GetTransposeNodeFromOutput.